src/main/java/com/es/phoneshop/exception/ProductDefinitionException.java

```diff
@@ -1,11 +1,18 @@
 package com.es.phoneshop.exception;
 
 public class ProductDefinitionException extends RuntimeException {
```

Comment:
1223

Reply:
1223123

Reply:
1223

Reply:
got it
then let's split this 2 cases with 2 exceptions: in case of nullable argument we will throw IllegalArgumentException, in case of not found product - your custom one